proxy/src/control/mod.rs

```diff
@@ -217,6 +217,7 @@ where
 
 // ===== impl  EnumService =====
 
+/*
```

Comment:
Could all this be deleted?

Reply:
Yeah, I don't think I touched that file enough to notice the big blob of commented out code. Will remove.